packages/db/prisma/client.ts

```diff
@@ -0,0 +1,22 @@
+import { PrismaClient } from "@prisma/client";
```

Comment:
Moving this logic to his own file was necessary to follow the vitest convention to mock modules.